core/kazoo_numbers/doc/knm_telnyx

```diff
@@ -0,0 +1,55 @@
+
+# knm_telnyx
+
+## Documentation
+* [Telnyx general documentation](https://developers.telnyx.com/docs/api/v1/numbers)
+
+
+## system_config
+#### Configuring number manager
+
+To enable kazoo number manager to use the telnyx module you will need to first update the `number_manager` document in the `system_config` database.
+
+In the configuration section that applies to your crossbar servers (or the default) add `"knm_voxbone"` to the `carrier_modules` parameter.
```

Comment:
Why Voxbone reference present in knm_telnyx documentation?

Reply:
Good question, let me remove that

Reply:
i investigated the situation, and i have come to the conclusion that it must have been a typo.